### ISSUE TITLE
Pr

### ISSUE THREAD

**Issue by Cinnamonsroll** [dennisfarmer/Ensemble-Video-Downloader#1]
Can i open a pr and fix some of the less than good code?

**Comment by dennisfarmer** [comment]
Yeah that'd be dope, I imagine you could use some sort of dictionary map thingy to make the conversion from pinyin to numeric pinyin less bad. I'll probs mess with it in the future when I'm not as busy and have time/brainspace to tackle chinese again lol
